### DISCUSSION TITLE
"The Mix manifest does not exist" (500 Server Error) after deployment laravel project to server.

### DISCUSSION THREAD

**Discussion by raisrindo** [austintoddj/canvas#989] [Show and tell]
My web works normally using canvas in the local computer but after I put it on server, it gives me 500 Server Error (when I set APP_ENV=production) or "The Mix manifest does not exist"(when I set APP_DEBUG=true). And then I got the clue from https://stackoverflow.com/questions/45153738/the-mix-manifest-does-not-exist-when-it-does-exist
and add :
$this->app->bind('path.public', function() {
        return base_path().'/../public_html';
 });
inside boot() method at AppServiceProvider file (under the app/Providers).
It works for me, I hope this information can help. This package is Awsome

**Comment by austintoddj** [comment]
Closing as obsolete with [Canvas v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0).

Admin assets are published via `php artisan canvas:publish` (or `canvas:install`); there is no Mix manifest step for Canvas itself. Host apps using Vite for their own frontend are unrelated to the old Mix error path.
